src/database/migrations/versions/2da0dc469be8_temp_migration.py

```
@@ -7,9 +7,8 @@
 """
 from typing import Sequence, Union
 
-from alembic import op
 import sqlalchemy as sa
-
+from alembic import op
```

This file uses `op.alter_column` to shrink token columns from length 255 to 64. That operation can fail or silently truncate existing values if any tokens are longer than 64 characters. Consider adding a pre-check (e.g. `SELECT COUNT(*) FROM table WHERE length(token) > 64`) and either trim those values intentionally or raise a clear error, or include an explicit data migration step to normalize/truncate tokens before altering the column.